www/src/components/layout.js

```diff
@@ -208,4 +211,18 @@ class DefaultLayout extends React.Component {
   }
 }
 
+const styles = {
+  skipLink: {
```

Comment:
Perhaps use Ryan's version?

https://ui.reach.tech/skip-nav

Reply:
I don’t understand the focus style 🤔 I took the styles from Shopify